js/components/Breakpoints.js

```diff
@@ -44,36 +52,12 @@ const Breakpoints = React.createClass({
 
   displayName: "Breakpoints",
 
-  onResumeClick() {
-    gThreadClient.resume();
-  },
-
-  onStepOverClick() {
-    gThreadClient.stepOver();
-  },
-
-  onStepInClick() {
-    gThreadClient.stepIn();
-  },
-
-  onStepOutClick() {
-    gThreadClient.stepOut();
-  },
-
   render() {
     return dom.div(
       { className: "breakpoints" },
-      dom.div(null,
-        dom.button({ onClick: this.onResumeClick }, "Resume"),
-        dom.button({ onClick: this.onStepOverClick }, "Over"),
-        dom.button({ onClick: this.onStepInClick }, "In"),
-        dom.button({ onClick: this.onStepOutClick }, "Out")
-      ),
-      dom.ul(
-        null,
-        this.props.breakpoints.valueSeq().map(bp => {
-          return renderBreakpoint(this.props.sources, bp);
-        })
```

Comment:
I'm really happy that in many places this PR cleaned up existing components. i.e. extracting actions from Breakpoints.